docs/02-get-started/05-observability.md

```diff
@@ -20,16 +20,40 @@ To do that, we'll use the Grafana dashboard that comes with Kyma.
 
 ## View the logs
 
-1. In the Grafana dashboard, use the left menu to navigate to **Explore** and choose **Loki** from the dropdown list.
+<div tabs name="View the logs" group="view-logs">
+  <details open>
+  <summary label="Grafana dasboard">
+  Grafana dashboard
+  </summary>
+
+1. Use the left menu to navigate to **Explore** and choose **Loki** from the dropdown list.
 2. Click on **Log browser** and select the following values:
    - **1. Select labels to search in**: `container`, `function`
-   - **2. Find values for the selected labels**: for **function** choose `lastorder`, for **container** choose `function` 
-   
-   Click **Show logs**.
+   - **2. Find values for the selected labels**: for **function** choose `lastorder`, for **container** choose `function`
+  
+3. Click **Show logs**.
 
     > **NOTE:** Alternatively, type or paste the `{function="lastorder", container="function"}` query and press `Shift`+`Enter` or click on **Run query**. 
 
-You can now browse the logs. 
+You can now browse the logs.
+
+  </details>
+  <details>
+  <summary label="Kyma Dashboard">
+  Kyma Dashboard
+  </summary>
+
+1. Go to the `default` Namespace.
+   
+2. Go to **Workloads** > **Functions**.
+   
+3. Select your `lastorder` Function.
+   
+4. Scroll down and click on **View Logs**.
+
+You can now browse the logs.
```

Comment:
1) Should it also be reflected in the [Observability tutorials](https://deploy-preview-13068--kyma-project-docs-preview.netlify.app/docs/kyma/preview/03-tutorials/00-observability/)?
Remember that Get Started is just an introductory flow, and we have more-in-depth tutorials that new information should be added to.